tests/docs-screenshots.spec.ts

```diff
@@ -68,43 +90,646 @@ test("instances", async ({ page }) => {
+
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_create.png",
+    clip: getClipPosition(240, 0, 1420, 750),
+  });
+
+  await page.getByPlaceholder("Enter value").fill("100");
+  await page.getByLabel("Select disk size unit").selectOption("MB");
+  await page.getByRole("button", { name: "Create", exact: true }).click();
+  await page.getByTestId("notification-close-button").click();
+  await page.getByRole("link", { name: volumeName }).click();
+  await page.waitForSelector(`text=Overview`);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_overview.png",
+    clip: getClipPosition(240, 0, 1420, 550),
   });
+
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_rename.png",
+    clip: getClipPosition(230, 0, 630, 40),
+  });
+
+  await page.getByLabel("Copy volume").click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_copy_modal.png",
+    clip: getClipPosition(445, 130, 1000, 680),
+  });
+  await page.getByLabel("Close active modal").click();
+
+  await page.getByRole("button", { name: "Export" }).click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_export.png",
+    clip: getClipPosition(445, 100, 1000, 710),
+  });
+  await page.getByLabel("Close active modal").click();
+
+  await deleteVolume(page, volumeName.toLowerCase());
+  await deletePool(page, poolName);
+});
+
+test("storage volume snapshots", async ({ page }) => {
+  const poolName = "pool1";
+  const volumeName = "CustomVol1";
+  const snapshot = "snapshot1";
+
+  await page.setViewportSize({ width: 1440, height: 800 });
+  await createPool(page, poolName);
+  await createVolume(page, volumeName);
+
+  await page.getByRole("link", { name: "Volumes", exact: true }).click();
+  await page.getByRole("link", { name: volumeName }).click();
+  await page.getByTestId("tab-link-Snapshots").click();
+  await expect(
+    page.getByText("There are no snapshots for this volume."),
+  ).toBeVisible();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_snapshots_tab.png",
+    clip: getClipPosition(240, 0, 1420, 350),
+  });
+
+  await page.getByRole("button", { name: "Create snapshot" }).click();
+  await page.getByLabel("Snapshot name").click();
+  await page.getByLabel("Snapshot name").fill(snapshot);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_snapshots_create.png",
+    clip: getClipPosition(480, 250, 950, 560),
+  });
+
+  await page.getByRole("button", { name: "Create snapshot" }).last().click();
+  await page.waitForSelector(
+    `text=Snapshot ${snapshot} created for volume ${volumeName}.`,
+  );
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/storage/storage_volumes_snapshots_list.png",
+    clip: getClipPosition(240, 0, 1420, 320),
+  });
```

Comment:
The snapshot is missing in the screenshot. This might need some logic to ensure the cache reloaded and the snapshot list is visible. On subsequent runs, the snapshot was visible.

<img width="1901" height="698" alt="image" src="https://github.com/user-attachments/assets/4338e89f-f3a1-4673-aa9f-e21575bc0877" />